### ISSUE TITLE
Outdated URLs to Threejs in website and MD files.

### ISSUE THREAD

**Issue by superjose** [aframevr/aframe#2733]
**Description:**
_I messed up and created a PR instead of an issue._

Anyways, here's the thing: There are a couple of URLs that point to the docs in the Threejs website that seem to be outdated. 

In here:
https://github.com/aframevr/aframe/blob/master/docs/components/material.md
https://aframe.io/docs/0.5.0/components/material.html#standard

Urls have this form:
https://threejs.org/docs/#Reference/Materials/ShaderMaterial

Threejs has this urls:
https://threejs.org/docs/#api/materials/ShaderMaterial

Apparently Threejs has renamed the #Reference for #api.

Thanks!

**Comment by rttomlinson** [comment]
I'll fix this.

**Comment by tejalavu96** [comment]
can you explain the issue clearly please

**Comment by superjose** [comment]
@tejalavu96 Yes. The URL pointing to Threejs in the A0Frame site yield a 404 error when clicked. Threejs has changed "#Reference" from the URL to "#api" 

In other words, try clicking any Threejs link in the A-Frame website that refers to Threejs.
